packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -39,16 +43,44 @@ const { Fail } = assert;
 const HolderI = M.interface('holder', {
   getPublicTopics: M.call().returns(TopicsRecordShape),
   makeDelegateInvitation: M.call(M.string(), AmountShape).returns(M.promise()),
+  makeWithdrawRewardInvitation: M.call(M.string()).returns(M.promise()),
```

Comment:
would be nice to add `withdrawReward` to this interface as well

Reply:
seems like a lot of redundancy: we have each method on 3 facets - twice on the `holder` facet if we keep that pattern up. Plus all the type declarations and interface guards.

Maybe the methods like `makeDelegateInvitation` on `holder` can just go away, since we have `Delegate` on the `invitationMakers` facet.

Reply:
given the positive feedback: 8b8f5bda4

It's slimmer: +56 -86